### DISCUSSION TITLE
Problem with get_report() function

### DISCUSSION THREAD

**Discussion by katycold** [zubkov-andrei/pg_profile#84] [Q&A]
Добрый день. установлен релиз 4.3 (последний). 
Все функции отрабатывают без ошибок. Но при выполнении функции get_report или функции get_report_latest отчет, который выгружается не содержит ни одной секции.

Postgres profile report (10 -14)
pg_profile version 4.3

Server name: local

Report interval: 2023-12-22 16:36:10+03 - 2023-12-25 11:51:40+03

Report sections

И все.

Не подскажете, в чём может быть проблема?

**Comment by zubkov-andrei** [comment]
Добрый день!
Как-то странно выглядит. А привидите, пожалуйста, полный текст команды, которой вы отчёт строите.

**Reply by katycold** [reply]
select profile.get_report_latest();

**Reply by zubkov-andrei** [reply]
Было бы интересно изучить что у вас там происходит. Можете ли вы передать [дамп наблюдений pg_profile](https://github.com/zubkov-andrei/pg_profile/blob/master/doc/pg_profile.md#data-export-and-import) для разбора? Нет ли там какой особой секретности?

**Reply by katycold** [reply]
Да, я могу сделать дамп

**Reply by zubkov-andrei** [reply]
У меня по вашему дампу отчёты строятся без проблем. Я попрошу вас попробовать построить отчёт используя обычный psql, как это указано в доке. Во всяком случае, я строил именно так. Ну, например:
```
psql -Aqtc "select profile.get_report(10,14)" -o report10_14.html
```
Если получится, надо искать где по пути теряются данные из функции построения отчёта, если не получится, будем разбираться с профайлом дальше.

**Reply by katycold** [reply]
С помощью psql отчет построился нормально. Таблицы на месте. Спасибо!
![image](https://github.com/zubkov-andrei/pg_profile/assets/154801932/ae7c1a0b-4237-4735-aee9-080153c56b8a)

**Comment by zubkov-andrei** [comment]
Также обратите внимание на #83. А то участились подобные случаи. Проверьте - не это ли у вас?

**Reply by katycold** [reply]
Кажется это не мой случай. Отчет формирую в pgadmin. да и весит отчет немного (220-400кб)
